ansible_collections/arista/avd/tests/unit/filters/test_range_expand.py

```diff
@@ -8,43 +8,95 @@
-    "Et1-2/3-4/5-6",
-    "65100.0",
-    "65100.0-4",
-    "65100.0-2,65200.1-2",
-    "1-2.0-1",
-]
-
-EXPECTED_RESULT_VALID_VALUES = [
-    ["Ethernet1"],
-    ["Ethernet1", "Ethernet2"],
-    ["Eth 3", "Eth 4", "Eth 5", "Eth 7", "Eth 8"],
-    ["et2", "et3", "et4", "et5", "et6", "po1", "po2"],
-    ["Ethernet1"],
-    ["Ethernet 1", "Ethernet 2", "Eth3", "Eth4", "Eth5", "7", "8"],
-    ["Ethernet2", "Ethernet3", "Ethernet4", "Ethernet5", "Ethernet6", "Port-channel1", "Port-channel2"],
-    ["Ethernet1/1", "Ethernet1/2", "Eth1/3", "Eth1/4", "Eth1/5", "Eth2/3", "Eth2/4", "Eth2/5", "Eth5/1", "Eth5/2"],
-    ["Eth1.1", "Eth9.1", "Eth10.1", "Eth2.2", "Eth2.3", "Eth3/1.3", "Eth3/1.4", "Eth3/2.3", "Eth3/2.4"],
-    ["1", "2", "3"],
-    ["1", "2", "3"],
-    ["vlan1", "vlan2", "vlan3"],
-    ["Et1/3/5", "Et1/3/6", "Et1/4/5", "Et1/4/6", "Et2/3/5", "Et2/3/6", "Et2/4/5", "Et2/4/6"],
-    ["65100.0"],
-    ["65100.0", "65100.1", "65100.2", "65100.3", "65100.4"],
-    ["65100.0", "65100.1", "65100.2", "65200.1", "65200.2"],
-    ["1.0", "1.1", "2.0", "2.1"],
+    ("Ethernet1", ["Ethernet1"]),
+    ("Ethernet1-2", ["Ethernet1", "Ethernet2"]),
+    ("Eth 3-5,7-8", ["Eth 3", "Eth 4", "Eth 5", "Eth 7", "Eth 8"]),
+    ("et2-6,po1-2", ["et2", "et3", "et4", "et5", "et6", "po1", "po2"]),
+    (["Ethernet1"], ["Ethernet1"]),
+    (["Ethernet 1-2", "Eth3-5", "7-8"], ["Ethernet 1", "Ethernet 2", "Eth3", "Eth4", "Eth5", "7", "8"]),
+    (["Ethernet2-6", "Port-channel1-2"], ["Ethernet2", "Ethernet3", "Ethernet4", "Ethernet5", "Ethernet6", "Port-channel1", "Port-channel2"]),
+    (["Ethernet1/1-2", "Eth1-2/3-5,5/1-2"], ["Ethernet1/1", "Ethernet1/2", "Eth1/3", "Eth1/4", "Eth1/5", "Eth2/3", "Eth2/4", "Eth2/5", "Eth5/1", "Eth5/2"]),
+    (["Eth1.1,9-10.1", "Eth2.2-3", "Eth3/1-2.3-4"], ["Eth1.1", "Eth9.1", "Eth10.1", "Eth2.2", "Eth2.3", "Eth3/1.3", "Eth3/1.4", "Eth3/2.3", "Eth3/2.4"]),
+    ("1-3", ["1", "2", "3"]),
+    (["1", "2", "3"], ["1", "2", "3"]),
+    ("vlan1-3", ["vlan1", "vlan2", "vlan3"]),
+    ("Et1-2/3-4/5-6", ["Et1/3/5", "Et1/3/6", "Et1/4/5", "Et1/4/6", "Et2/3/5", "Et2/3/6", "Et2/4/5", "Et2/4/6"]),
+    ("65100.0", ["65100.0"]),
+    ("65100.0-4", ["65100.0", "65100.1", "65100.2", "65100.3", "65100.4"]),
+    (
+        "65100.0-2,65200.1-2",
+        ["65100.0", "65100.1", "65100.2", "65200.1", "65200.2"],
+    ),
+    ("1-2.0-1", ["1.0", "1.1", "2.0", "2.1"]),
+    ("Ethernet(1)", ["Ethernet1"]),
+    ("Ethernet(1,2,3)", ["Ethernet1", "Ethernet2", "Ethernet3"]),
+    ("Ethernet(1)/(4,5)", ["Ethernet1/4", "Ethernet1/5"]),
+    ("(1,2)", ["1", "2"]),
+    ("1,(2,3)/5", ["1", "2/5", "3/5"]),
+    ("Eth(10-14)", ["Eth10", "Eth11", "Eth12", "Eth13", "Eth14"]),
+    ("Eth(101-103)/(5,7)", ["Eth101/5", "Eth102/5", "Eth103/5", "Eth101/7", "Eth102/7", "Eth103/7"]),
+    (
+        "Eth(1,2",
+        ["Eth(1", "Eth2"],
+    ),
+    # ("Eth(1,2)/3,4)", ["Eth1/3", "Eth2/3", "Eth4)"]), # Exception need to fix regex_commas_outside_parentheses regex in range_expand
```

Comment:
Has this been addressed?

Reply:
I have not found the solution for this yet, so commented the testcase to notify.

Reply:
We should change these invalid tests to use { and }

Reply:
Done